src/document/url.ml

```diff
@@ -377,6 +384,9 @@ module Anchor = struct
     | { iv = `SourcePage (p, _name); _ } | { iv = `SourceDir (p, _name); _ } ->
         let page = Path.from_identifier (p :> Path.source) in
         Ok { page; kind = `Page; anchor = "" }
+    | { iv = `AssetFile (p, _name); _ } ->
+        let page = Path.from_identifier p in
+        Ok { page; kind = `File; anchor = "" }
```

Comment:
I copied what was done for ``` `SourcePage ``` here, but honestly it looks weird.

Reply:
> Is it the `anchor = ""` which is bothering you?

No, it's that we're getting a url to the parent of the asset, not the asset itself.

Reply:
Indeed, that's a bug, thanks for spotting it! It was not harmful because We only call `Path.from_identifier` on the `` `SourcePage`` variant, never `Anchor.from_identifier`.

```suggestion
    | { iv = `AssetFile _; _ } as p ->
        let page = Path.from_identifier p in
        Ok { page; kind = `File; anchor = "" }
```

Could you correct `SourcePage` and `SourceDir` as well?